assets-src/styles/sass/30-base/_links.scss

```diff
@@ -48,4 +47,27 @@ a.with-icon--after {
 		color: $black;
 		outline-width: 0; /* 2 */
 	}
+}
+
+.heading-anchor {
+    @extend a, :not([class]);
+
+    color: $storm-gray;
+    font-weight: normal;
+    //margin-inline-start: 0.25em;
+    opacity: 0.82;
+    text-underline-offset: auto;
+
+    &:visited {
+        color: $storm-gray;
+    }
+
+    &:hover {
+        opacity: 1;
+    }
```

Comment:
I think because of these lines, hovering a visited anchor is gray, while hovering a non-visited one is blue. Is that the expected behavior?

Reply:
@NicolaSaunders I indeed wondered if `color: $storm-gray;` was missing for hover.

Reply:
Apologies and well spotted - I have pushed up an amendment for this.